content/en/docs/setup/additional-setup/compatibility-versions/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Compatibility Versions
+description: How to configure "compatibility versions", to decouple behavioral changes from releases.
+weight: 36
+keywords: [profiles,install,helm]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+With each new version of Istio, there may be some intentional behavioral changes.
+These can be to improve security, fix incorrect behavior, or otherwise improve Istio for users.
+Generally, these types of changes impact only edge cases
+
+While beneficial on the long term, each behavioral change introduces risk for during upgrades.
+Historically, when upgrading users should read the release notes for any behavioral changes and determine if they are impacted; this can be tedious and error prone.
+
+Compatibility versions give users an additional option, allowing release versions to be decoupled from behavioral changes.
+For instance, you can install Istio {{< istio_version >}}, but configure it to behave like {{< istio_previous_version >}}.
+
+## Using compatibility versions
+
+To use a compatibility version, simple set the `compatibilityVersion` field.
+
+{{< tabset category-name="install" >}}
+{{< tab name="IstioOperator" category-value="iop" >}}
+
+{{< text shell >}}
+$ helm install ... --set compatibilityVersion={{< istio_previous_version >}}
+{{< /text >}}
+
+{{< /tab >}}
+{{< tab name="Helm" category-value="helm" >}}
+
+{{< text shell >}}
+$ istioctl install --set values.compatibilityVersion={{< istio_previous_version >}}
+{{< /text >}}
+
+{{< /tab >}}
+{{< /tabset >}}
+
+## When should I use compatibility versions?
+
+Compatibility versions are recommended to be used only when an incompatibility is found, rather than as the default.
+Each compatibility version will only persist for a few releases, so eventually you will need to migrate to the new behavior.
```

Comment:
Wondering if we need a general rule how long these should persist? Deprecation rule?